src/styles/generalStyles.css

```
@@ -424,3 +424,7 @@ html.transitioning-color-scheme * {
 .gatsby-image-wrapper-rounded img {
   @apply rounded-full;
 }
+
+img {
```

i think we should scope this to the markdown, lmk if images outside the markdown also break in dark mode?
```suggestion
.markdown img {
```